### PR TITLE
Add Mongo storage solution

### PR DESCRIPTION
**Pull request by filipecabaco**
First step to replace Cassandra as our main entry storage.

### CODE REVIEW

**Comment by jvvasques** [comment]
Looks good to me. A docker image would be nice to have 👍  what do you say @nmcc ?

**Comment by filipecabaco** [comment]
already doing that, almost there fully replacing cassandra but I'm having problem with _id's repeated. 

I need to recheck what I'm doing but it should be quick 👍

File: talkpocket-api/src/talkpocket_api/storage/mongo.clj [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+(defn- keywordize [input]
+  (from-db-object input true))
+
+(defn find
+  ([] (map keywordize (coll/find-maps @db collection)))
+  ([id] (keywordize (coll/find-map-by-id @db collection (ObjectId. id)))))
+
+(defn consumer
+  "Consumer that receives a map and persists it to Cassandra"
```
Comment (nmcc) [inline talkpocket-api/src/talkpocket_api/storage/mongo.clj]: `persists it to Cassandra`?? 🤔

**Review by nmcc — COMMENTED** [review]
Quite simple change, indeed

**Comment by nmcc** [comment]
Docker FTW!!

File: talkpocket-api/src/talkpocket_api/storage/mongo.clj (hunk 1 above)
Comment (filipecabaco) [reply talkpocket-api/src/talkpocket_api/storage/mongo.clj]: still doing those changes, this was just to showcase mongo in clojure

**Comment by filipecabaco** [comment]
I'm actually tempted to remove docker stuff and use startup scripts... less prone to error.

Just keep the services dockerized

File: talkpocket-api/src/talkpocket_api/storage/mongo.clj (hunk 1 above)
Comment (jvvasques) [reply talkpocket-api/src/talkpocket_api/storage/mongo.clj]: :trollface:

**Comment by filipecabaco** [comment]
Still has problems with minio, almost there solving that one, after that I'm going to check for a better way to generate id's that follow the idea of URL and language as a defining element of ID generation but also following a similar idea to https://blog.twitter.com/2010/announcing-snowflake

**Pull request merged**